### PR TITLE
Added Overall Velocity CDF visualization

### PR DESCRIPTION
**Pull request by youralien**
Summary:
created new method in visualization.py for generating the cdf of velocities.
added thinkplot and thinkstats2 to the repo as helper libraries.
removed extra imports in visualization.py.

Tests:
To show the plot, I ran
python visualization.py {PATHTO}/stmarc.sqlite 30 {PATHTO}/homography.txt --vis-type=vel-overall-cdf

### CODE REVIEW

File: trafficcloud/plotting/thinkplot.py [hunk 1]
```diff
@@ -0,0 +1,781 @@
...
+
+
+def main():
+    color_iter = _Brewer.ColorGenerator(7)
+    for color in color_iter:
+        print(color)
+
+
+if __name__ == '__main__':
+    main()
```
Comment (runnersaw) [inline trafficcloud/plotting/thinkplot.py:780]: I read none of this file

File: trafficcloud/plotting/visualization.py [hunk 2]
```diff
@@ -288,6 +283,74 @@ def vel_histograms(fig, filename, fps, vistype='overall'):
...
+    obj_id = 0;
+    obj_vels = [];
+
+    xvels = []
+    yvels = []
+    for row in cursor:
+        xvel = row[4]
+        yvel = row[5]
+
+        mph = 2.23694
```
Comment (runnersaw) [inline trafficcloud/plotting/visualization.py]: Maybe comment why this is. (I think this is mph/(m/s), is that right?)

**Review by runnersaw — APPROVED** [review]

**Review by dcelik — APPROVED** [review]
Looks good to me, we should add attribution to Allen if we decide to keep using thinkplot.

**Comment by youralien** [comment]
for the issue on the MPH conversion, I made it a global that works for all the meters-per-second to miles-per-hour conversions

```
# 2.23694 mph / (m/s) = (1 mi / 1609.34 m) * (3600 sec / 1 hr)
MPS_MPH_CONVERSION = 2.23694
```

**Pull request merged**